features/DocumentController.feature

```diff
@@ -173,6 +173,25 @@ Feature: Document Controller
       | _id          |
       | "document-1" |
 
+  # document:deleteFields ===========================================================
+  @mappings
+  Scenario: Update document with and without returning updated document
```

Comment:
It's seem's to me that the name of the scenario does not reflect what done inside.
Also, they should be split into two distinct use case.